docs/compression_algorithms/Quantization.md

```diff
@@ -310,116 +310,96 @@ Following is an example of wrapping ImageNet validation loop as a callback. Top5
-    "export_to_onnx_standard_ops": false,
+    "target_device": "TRIAL"
 }
 ```
 
-***Per layer ranges initializations parameters***:
-Per layer ranges initiaization can be enabled by specifying  in `"initializer"` section `"range"` as list of dictionaries in the following format:
-
-```
+- Quantize a multi-input model to 8-bit precision targeted for Intel CPUs, with a range initialization performed using percentile statistics (empirically known to be better for NLP models, for example) and excluding some parts of the model from quantization:
+```json5
 {
-    "range": [
-        {
-            "type": "min_max", // Type of the initializer - determines which statistics gathered during initialization will be used to initialize the quantization ranges for all modules specified by `"target_scopes"` or `"ignored_scopes"`.
-
-            "num_init_samples": 256, // Number of samples from the training dataset to consume as sample model inputs for purposes of setting initial minimum and maximum quantization ranges
-
-            "target_scopes": [], // A list of model control flow graph node scopes to be considered for this operation - functions as a 'allowlist'. Optional.
-            "ignored_scopes": [], // A list of model control flow graph node scopes to be ignored for this operation - functions as a 'denylist'. Optional.
-            "target_quantizer_group": "weights" // Type of quantizer group to which this initialization of ranges will be applied. Optional. (By default this initialization of ranges will be applied to weights and activations quantizers)
+    "input_info": [
+       {
+            "keyword": "input_ids",
+            "sample_size": [1, 128],
+            "type": "long",
+            "filler": "ones"
         },
-        ...
-    ]
+        {
+            "keyword": "attention_mask",
+            "sample_size": [1, 128],
+            "type": "long",
+            "filler": "ones"
+        }
+    ], // the input shape of your model may vary
+    "compression": {
+       "algorithm": "quantization",
+       "initializer": {
+          "range": {
+             "num_init_samples": 64,
+             "type": "percentile",
+             "params": {
+                "min_percentile": 0.01,
+                "max_percentile": 99.99
+             }
+          }
+       },
+       "ignored_scopes": ["{re}BertSelfAttention\\[self\\]/__add___0",
+            "RobertaForSequenceClassification/RobertaClassificationHead[classifier]/Linear[out_proj]",
+            "RobertaForSequenceClassification/RobertaClassificationHead[classifier]/Linear[dense]"
+        ],
+    },
+    "target_device": "TRIAL"
 }
-
 ```
-Initialization of ranges defined in this way must specify an unambiguous initialization rule for each module.
+
```

Comment:
@ljaljushkin please look this through and advise whether we need to add a precision-init related config example here and in what capacity (so that the example is brief, but meaningful)

Reply:
I think this config is the most compact and meaningful to showcase. All algorithm-specific options are specified above and have some default values.
```json
"initializer": {
    "precision": {
        "type": "autoq", # or "type": "hawq"
        "bits": [2, 4, 8],
        "compression_ratio": 0.15,
        "iter_number": 300,
}
```

Reply:
Done